src/main/java/com/telefonica/iot/perseo/EventsServlet.java

```diff
@@ -76,25 +77,27 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
         Utils.putCorrelatorAndTrans(request);
         logger.debug("events doPost");
         response.setCharacterEncoding("UTF-8");
-        response.setContentType("application/json;charset=UTF-8");
-        PrintWriter out = response.getWriter();
-        try {
-            StringBuilder sb = new StringBuilder();
+        response.setContentType("application/json;charset=UTF-8");        
+        try {        	
             String eventText = Utils.getBodyAsString(request);
-            logger.info("incoming event:" + eventText);
+            logger.info(String.format("incoming event: %s", eventText));
             org.json.JSONObject jo = new JSONObject(eventText);
-            logger.debug("event as JSONObject: " + jo);
+            logger.debug(String.format("event as JSONObject: %s", jo));
             Map<String, Object> eventMap = Utils.JSONObject2Map(jo);
-            logger.debug("event as map: " + eventMap);
+            logger.debug(String.format("event as map: %s" , eventMap));
             epService.getEPRuntime().sendEvent(eventMap, Constants.IOT_EVENT);
-            logger.debug("event was sent: " + eventMap);
-        } catch (JSONException je) {
-            logger.error("error: " + je);
-            response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
-            out.printf("{\"error\":\"%s\"}\n", je.getMessage());
 
-        } finally {
-            out.close();
+            logger.debug(String.format("event was sent: %s", eventMap));
+        } catch (Exception je) {
+        	try {        		
```

Comment:
Please review indentation on this block. Have you done a checkstyle pass (configured using telefonica_checkstyle.xml file in the root of the repository) after the changes?

Reply:
Fixed in d3cd3f4c88c67b3621b6714e4af99b3048ad605d